d3a_api_client/utils.py

```diff
@@ -166,6 +166,26 @@ def get_area_uuid_and_name_mapping_from_simulation_id(collab_id, domain_name):
         return area_name_uuid_map
 
 
+def get_aggregators_list(domain_name):
+    """
+    Returns a list of aggregators for the logged in user
+    """
+    jwt_key = retrieve_jwt_key_from_server(domain_name)
+    from sgqlc.endpoint.http import HTTPEndpoint
```

Comment:
Any specific reason for lazy import?

Reply:
@faizan2590 There was no need, it was used several places in the file so I did some refactoring now